.circleci/config.yml

```diff
@@ -201,7 +201,7 @@ jobs:
           name: 🧪 Run Integration Tests
           command: |
             . venv/bin/activate && cd packages && ls -la
-            find . -name "*.gz" | xargs pip install --force-reinstall && pip list | grep dash && cd ..
+            find . -name "*.gz" | xargs pip install --ignore-installed && pip list | grep dash && cd ..
```

Comment:
the real problem is this option.   `--force-reinstall` claims to uninstall the old package, but looks like -I  or --ignore-installed should be the preferred way to go